src/index.html

```diff
@@ -17,11 +17,417 @@
+              href="#gadgets"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#tablets"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#photo"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#video"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="images/imac.jpeg"
+          class="card__image"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+        />
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__review">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__count-review">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-label">Price:</p>
+
+          <p class="card__price-value">$2,199</p>
+        </div>
+
+        <a
+          href="#buy"
+          class="card__button"
+          data-qa="card-hover"
+        >
+          Buy
+        </a>
+      </div>
+
+      <div class="card">
+        <img
+          src="images/imac.jpeg"
+          class="card__image"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+        />
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__review">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__count-review">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-label">Price:</p>
+
+          <p class="card__price-value">$2,199</p>
+        </div>
+
+        <a
+          href="#buy"
+          class="card__button"
+        >
+          Buy
+        </a>
+      </div>
+
+      <div class="card">
+        <img
+          src="images/imac.jpeg"
```

Comment:
The src attribute for the image uses a backslash (\) instead of a forward slash (/). This might cause issues in some environments. It's recommended to use forward slashes for file paths in HTML.